Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "parity-codec"
 description = "Lightweight, efficient, binary serialization and deserialization codec"
-version = "4.1.1-alpha.0"
+version = "4.1.2"
```

Comment:
Is version update ok? `alpha` stuff was added automatically the last time I released with `cargo-release` (tip of the day: just use `cargo publish` from small updates).

Reply:
Did we release `4.1.1`?

Reply:
Right. must be `4.1.1` now.